mount/mount_windows.go

```diff
@@ -17,17 +17,45 @@
 package mount
 
 import (
+	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/Microsoft/go-winio/pkg/bindfilter"
 	"github.com/Microsoft/hcsshim"
+	"github.com/containerd/containerd/log"
+	"golang.org/x/sys/windows"
 )
 
+const sourceStreamName = "containerd.io-source"
+
+var (
+	// ErrNotImplementOnWindows is returned when an action is not implemented for windows
+	ErrNotImplementOnWindows = errors.New("not implemented under windows")
+)
+
+func (m *Mount) ReadOnly() bool {
```

Comment:
I think we should not export this one because we don't export this in other platform.

Reply:
Will change this tomorrow morning.

Reply:
Would it be acceptable if I export it for all platforms? It simply looks for the ```ro``` option in a string array. If it's too specific, I could have it accept a parameter and rename it to ```HasOption()``` or similar. What do you think?

Reply:
Moved it to ```mount.go```. It's currently being used in the ```mount``` package and in the ```diff``` package. Let me know if you prefer another solution.